### PR TITLE
Export proc.cmdline

### PR DESCRIPTION
**Pull request by monicasarbu**
Containing the command line used to start the process, together with
the arguments separated by space.

WARNING: Please don't merge it yet as it doesn't work yet on Windows.

### CODE REVIEW

**Comment by monicasarbu** [comment]
The changes in gosigar are tracked here: https://github.com/elastic/gosigar/pull/1

File: etc/topbeat.local.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+################### Topbeat Configuration Example #########################
+
+############################# Input ############################################
+input:
```
Comment (ruflin) [inline etc/topbeat.local.yml]: should we take the opportunity and rename it top tobeat?

**Comment by ruflin** [comment]
@monicasarbu What is the goal of including a local config file?

**Comment by monicasarbu** [comment]
@ruflin I added the local file by mistake from my Windows machine. I am deleting it. Thanks for noticing it.

**Comment by tsg** [comment]
Needs to be reopened in the new merged repo.

**Comment by xrayrey** [comment]
Is this issue going to be reopened on the new merged repo? Also, would this feature be available for Linux on the next release? This is an extremely useful feature to debug applications across multiple servers.

**Comment by monicasarbu** [comment]
@xrayrey Yes, we are considering exporting proc.cmdline for Unix systems at least. I created another issue in the beats repo: https://github.com/elastic/beats/issues/533

**Comment by xrayrey** [comment]
@monicasarbu Awesome! Thanks!

**Pull request closed without merging**